lib/hpcbase.pm

```diff
@@ -19,5 +19,25 @@ sub enable_and_start {
     systemctl "start $arg";
 }
 
+sub upload_service_log {
+    my ($self, $service_name) = @_;
+    testapi::script_run("journalctl -u $service_name > /tmp/$service_name");
+    testapi::script_run("cat /tmp/$service_name");
+    testapi::upload_logs("/tmp/$service_name");
+}
+
+sub post_fail_hook {
+    my ($self) = @_;
+    testapi::script_run("journalctl -o short-precise > /tmp/journal.log");
+    testapi::script_run('cat /tmp/journal.log');
+    testapi::upload_logs('/tmp/journal.log');
+    hpcbase::upload_service_log('wickedd-dhcp4.service');
+    if (testapi::check_var('HPC', 'conman')) {
+        hpcbase::upload_service_log('conman');
+    }
+    elsif (testapi::check_var('HPC', 'powerman')) {
+        hpcbase::upload_service_log('powerman');
+    }
+}
+
 1;
-# vim: set sw=4 et:
```

Comment:
Is there reason to remove this line? It was removed from other files, but not from all of them,

Reply:
generally I am always removing them because I think that editor specific thing should stay on developer's computer and not in git. I think I remove it from all files which I edit , but I don't plan to remove them from whole project .